src/test/groovy/com/github/vkorobkov/jfixtures/integration/AutoGeneratedPkIntegrationTest.groovy

```diff
@@ -0,0 +1,57 @@
+package com.github.vkorobkov.jfixtures.integration
+
+import com.github.vkorobkov.jfixtures.testutil.H2Test
+import spock.lang.Specification
+
+import java.nio.file.Path
+
+class AutoGeneratedPkIntegrationTest extends Specification implements H2Test {
```

Comment:
Is there a test which checks that table A refers(by FK) to table B and table B has a custom Pk column name?